src/windows/title_editor.py

```diff
@@ -540,7 +540,7 @@ def set_font_style(self):
             s = text_child.attributes["style"].value
             ard = style_to_dict(s)
             set_if_existing(ard, "font-style", self.font_style)
-            set_if_existing(ard, "font-family", self.font_family)
+            set_if_existing(ard, "font-family", f"'{self.font_family}'")
```

Comment:
The old code wasn't doing this either, and it admittedly _seems_ to work fine without it, but technically font family names with a space are supposed to be inside single quotes, in CSS. (No real harm in doing it for all font family names, even one-word ones.)